src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,120 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[CAPACITY];
+
+    }
 
     @Override
     public void put(K key, V value) {
+        if (size == table.length * LOAD_FACTOR) {
+            resize();
+        }
+        int index = getPositionInTable(key);
+        Node<K, V> current = table[index];
+
+        if (current == null) {
+            table[index] = new Node<>(key, value, null);
+        }
 
+        while (current != null) {
+
+            if (Objects.equals(current.getKey(), key)) {
+                current.setValue(value);
+                return;
+            }
+
+            if (current.getNext() == null) {
+                Node<K, V> newNode = new Node<>(key, value, null);
+                current.setNext(newNode);
+                break;
+            }
+            current = current.getNext();
+        }
+        size++;
```

Comment:
Let's remove second if statement and move inner field from it to the bottom under while loop due to the fact `(current == null)` is a default case so that it'll work the same. And the same we should remove all redundant blank lines
```suggestion
        while (current != null) {
            if (Objects.equals(current.getKey(), key)) {
                current.setValue(value);
                return;
            }
            if (current.getNext() == null) {
                Node<K, V> newNode = new Node<>(key, value, null);
                current.setNext(newNode);
                break;
            }
            current = current.getNext();
        }
        table[index] = new Node<>(key, value, null);
        size++;
```